src/CancelRequestDialog.test.js

```diff
@@ -0,0 +1,113 @@
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../test/jest/__mock__';
+
+import CancelRequestDialog from './CancelRequestDialog';
+
+const labelIds = {
+  modalLabel: 'ui-requests.cancel.modalLabel',
+  modalText: 'ui-requests.cancel.requestWillBeCancelled',
+  cancelRequestDialogCancel: 'stripes-core.button.confirm',
+  cancelRequestDialogClose: 'stripes-core.button.back',
+};
+const testIds = {
+  cancelRequestDialog: 'cancelRequestDialog',
+  cancelRequestDialogCancel: 'cancelRequestDialogCancel',
+  cancelRequestDialogClose: 'cancelRequestDialogClose',
+};
+
+describe('CancelRequestDialog', () => {
+  const onCancelRequest = jest.fn();
+  const onClose = jest.fn();
+  const cancellationReasons = {
+    records: [{
+      name: 'cancellation reasons',
+      requiresAdditionalInformation: false,
+    }, {
+      name: 'other cancellation reasons',
+      requiresAdditionalInformation: true,
+    }],
+  };
+  const defaultProps = {
+    request: {
+      id: 'ad038163-7368-44e8-9057-c21bd2b9cd4c',
+      instance: {
+        title: 'title',
```

Comment:
You can another test that verifies if the modal displays the instance title as expected from props.request.instance.title

Reply:
Won't do. Hello we send title to formatted message.

Reply:
Fixed